components/ecoindex/config/settings.py

```diff
@@ -8,9 +10,10 @@ class Settings(BaseSettings):
     CORS_ALLOWED_ORIGINS: list = ["*"]
     DAILY_LIMIT_PER_HOST: int = 0
     DATABASE_URL: str = "sqlite+aiosqlite:///db.sqlite3"
+    DOCKER_CONTAINER: bool = False
     DEBUG: bool = False
     ENABLE_SCREENSHOT: bool = False
-    EXCLUDED_HOSTS: list[str] = ["localhost", "127.0.0.1"]
+    EXCLUDED_HOSTS: List[str] = ["localhost", "127.0.0.1"]  # Update the type annotation
```

Comment:
**🎯 suggestion(non-blocking):** From python 3.9 you can use directly `list`: https://stackoverflow.com/a/62033243 No need of `typing.List` as the project requires python 3.10+